### DISCUSSION TITLE
Scheduling and publishing calendar episodes ?

### DISCUSSION THREAD

**Discussion by rivadoza** [PodcastGenerator/PodcastGenerator#740] [Q&A] [answered]
Bonjour,

Thank you for this application, I find it very flexible and easy to use.

I am looking for a scheduling feature each day, an episode is specific to a day eg. the March 1st episode is automatically published on March 1st, then March 2nd for the March 2nd episode, etc. (with the management of February 29) and the auto update of the rss feed.

This would be automatic and configured once for all, the episode files will remain on the server permanently (except in the case of a modification of the file which would have to be replaced).

Did anyone having needed this before, or knowing a solution?

Best regards.

**Comment by charabaruk** [comment, marked as answer]
If you publish an episode with a date/time in the future, Podcast Generator will not display that episode on the public side until the first cron.php invocation after that publish time. So as long as you make sure that the cron setup for your site works, you can schedule episodes in advance directly in PG.
